src/components/RoomView/Winner.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useState } from 'react';
+import { setCurrentClue } from '../../store/clueSlice';
+import { useDispatch } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import { database } from '../../utils/firebase';
+import { ref, get, push, update, onValue } from 'firebase/database';
+import { useSelector } from 'react-redux';
+import { Button } from '@mui/material';
+import { setRoomId } from '../../store/playerSlice';
+interface ReduxState {
+    player: { playerId: string };
+    game: { status: string; winner: string };
+  }
+  const Winner:React.FC= () => {
+    const { roomId } = useParams();
+    setRoomId(roomId);
+    const playerId= useSelector((state: ReduxState) => state.player.playerId)
```

Comment:
This needs to use RootState, not ReduxState. The RootState is just export from the Redux store so need to define it here

Reply:
Heres an example:
<img width="663" alt="Screen Shot 2023-02-04 at 2 13 30 PM" src="https://user-images.githubusercontent.com/79232977/216791717-6c1bd3e6-cdac-4596-b779-b9f8f170932f.png">

Reply:
we can wait for the front end ts merged to deal with this pull request